AllisonsCatPalaceTDDTests/SwipeOverlayViewTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  SwipeOverlayViewTests.swift
+//  AllisonsCatPalaceTDD
+//
+//  Created by Joe Susnick on 8/27/17.
+//  Copyright © 2017 Joesus. All rights reserved.
+//
+
+@testable import AllisonsCatPalaceTDD
+import XCTest
+import Koloda
+
+class SwipeOverlayViewTests: XCTestCase {
+    var swipeOverlayView: SwipeOverlayView!
+
+    override func setUp() {
+        super.setUp()
+
+        let bundle = Bundle(for: SwipeOverlayView.self)
+        guard let view = bundle.loadNibNamed("SwipeOverlayView", owner: SwipeOverlayView().self)?.first as? SwipeOverlayView else {
+            return XCTFail("Should be able to instantiate SwipeOverlayView from nib")
+        }
+
+        swipeOverlayView = view
+    }
+
+    func testHasImageView() {
+        XCTAssertNotNil(swipeOverlayView.imageView,
```

Comment:
guard let the imageView then test that the imageView's image is nil and check whatever else I need to about the imageView